pb-rs/src/types.rs

```diff
@@ -1021,6 +1026,57 @@ impl Message {
         Ok(())
     }
 
+    fn write_impl_rentals<W: Write>(&self, w: &mut W) -> Result<()> {
+        write!(w,
```

Comment:
Note: I found the single `write!` to be easier to code and parse visually than the `write!`-per-line that is common in this file.